tests/telemetry/test_telemetry.py

```diff
@@ -270,9 +270,6 @@ def test_handled_integration_error(test_agent_session, run_python_code_in_subpro
 
 def test_unhandled_integration_error(test_agent_session, ddtrace_run_python_code_in_subprocess):
     code = """
-import logging
-logging.basicConfig()
-
```

Comment:
Are we just removing this because we don't need 2 integrations here?

Reply:
The ddtrace logger is configured in `ddtrace/__init__.py`. Calling basicConfig here does nothing, so we can remove this code.